frontend/packages/dev-console/src/components/projects/CreateProjectListPage.tsx

```diff
@@ -17,24 +15,16 @@ const CreateProjectListPage: React.FC<CreateProjectListPageProps> = ({
   children,
   ...props
 }) => {
-  const { t } = useTranslation();
   const [, setActiveNamespace] = useActiveNamespace();
   const handleSubmit = (project: K8sResourceKind) => {
     setActiveNamespace(project.metadata?.name);
     onCreate && onCreate(project);
   };
-
   const openProjectModal = () => createProjectModal({ blocking: true, onSubmit: handleSubmit });
-
   return (
-    <Trans t={t} ns="devconsole">
-      <ProjectListPage {...props} title={title}>
-        {children} {'or '}
-        <Button isInline variant="link" onClick={openProjectModal}>
-          {'create a Project.'}
-        </Button>
-      </ProjectListPage>
-    </Trans>
+    <ProjectListPage {...props} title={title}>
+      {typeof children === 'function' ? children(openProjectModal) : children}
```

Comment:
Please type the props accordingly.

Reply:
have updated , PTAL https://github.com/openshift/console/compare/489dec3448aa728bc655a4fdc235ccc56a195e37..7e5dd40d7ba1e5f6bc78b0af5cfb942873ad4f14